CONTRIBUTING.md

```diff
@@ -13,26 +24,20 @@ You can find our code of conduct [here](CODE_OF_CONDUCT.md).
 
 ### Commits
 
-Please follow our simple commit style guidelines.
+Don't be afraid to have a bunch of commits while working on a pull-request. We end up
+squashing all of them before merging to the `master` branch anyways.
 
-1. Prepend all commits with a topic.
-   Available topics are:
-    - `[storage]` For changes to `ink_storage`.
-    - `[env]` For changes to `ink_env`.
-    - `[lang]` For changes to `ink_lang`.
-    - `[allocator]` For changes to `ink_allocator`.
-    - `[metadata]` For changes to `ink_metadata`.
-    - `[prelude]` For changes to `ink_prelude`.
-    - `[primitives]` For changes to `ink_primitives`.
-    - `[ci]` For changes to the continuous integration.
-    - `[chores]` For general non-technical changes.
-    - `[examples]` For changes to the examples.
+We'd appreciate it if kept your commits small and descriptive. It makes the review
+process easier - which means you get a 🟩 from Github sooner (that's why you're
+contributing in the first place anyways, right?)
 
-For other purposes and sections please try to find a good fit for the topic.
+To help you out here's a [really good post](https://cbea.ms/git-commit/) on how to write good commit
+messages.
 
 ### Checklist
 
 Below is a checklist for you before doing a pull request.
+
 Following these will ensure that your pull request is going to be accepted.
```

Comment:
Wow this checklist below is super exhaustive, I don't think anybody will ever go through the whole thing. We should simplify it as well. 

There are two `scripts/check-*` scripts to do a number of these things of this locally, could you check if they are referenced somewhere in here and still working? We could refer to them instead.

Reply:
Let's do it in a follow-up.